src/selectors/generateChannelLink.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Returns a link depending on the channel
+ * Depending on pageUrl, will either append to existing query params or create a new query
+ * @param {string} pageURL - Page url.
+ * @param {string} integrationName - Will append channel depending on integration name.
+ */
+
+export function generateChannelLink(pageURL, integrationName) {
```

Comment:
Talk to Garrett and see if we can rework the channel values so they can be generated dynamically.  I want to avoid hard coding channel values so we don't have to modify code to add new ones.